### ISSUE TITLE
full text search resets column order

### ISSUE THREAD

**Issue by fossildoc** [dbeaver/dbeaver#10340]
#### System information: 
-Ubuntu 20.04.1
- DBeaver version
7.2.4
- Java version 
1.8.0_272
- Additional extensions

#### Connection specification:
- Database name and version
postgresql
- Driver name
jdbc:postgresql
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
no
#### Describe the problem you're observing:
When a full text search is made, any tables in which there is a hit will undo user-specified column visibiity and/or ordering. The button "remove all filters/orderings" should only remove filters. There should be a separate button to remove orderings. Even if that button is not clicked, clearing the select statement just above the list of hits will also undo all the orderings.

#### Steps to reproduce, if exist:
Click "view data" for several tables in a db. Rearrange the column ordering in each of them. Do a full text search of all tables for something that is likely to be found in at least one table. Close the results window. Try, by any method, to restore the full display of the table(s) in which a hit was found. Result: the column ordering will be undone.

**Comment by uslss** [comment]
thanks for the report

**Comment by ShadelessFox** [comment]
I don't think this is really a bug rather a feature request to separate button functionality. I'll change labels.
